test/widgets/autocomplete_test.dart

```diff
@@ -65,10 +74,25 @@ void main() {
   TestZulipBinding.ensureInitialized();
 
   group('ComposeAutocomplete', () {
+
+    Finder findNetworkImage(String url) {
+      return find.byWidgetPredicate((widget) => switch(widget) {
+        Image(image: NetworkImage(url: var imageUrl)) when imageUrl == url => true,
+        _ => false,
```

Comment:
nit: because the first pattern is so long, the result "true" gets a bit hard to spot when it's off at the end of that line. So:
```suggestion
        Image(image: NetworkImage(url: var imageUrl)) when imageUrl == url
          => true,
        _ => false,
```